build/source/driver/summa_restart.f90

```diff
@@ -227,6 +230,20 @@ subroutine summa_readRestart(summa1_struc, err, message)
    dt_init%gru(iGRU)%hru(iHRU) = progStruct%gru(iGRU)%hru(iHRU)%var(iLookPROG%dt_init)%dat(1) ! seconds
   end do
 
+  ! *****************************************************************************
+  ! *** initialize HDS variables at the GRU level
+  ! *****************************************************************************
+  if(model_decisions(iLookDECISIONS%prPotholes)%iDecision == HDSmodel)then
+   call init_summa_HDS(bvarStruct%gru(iGRU)%var(iLookBVAR%pondVolFrac)%dat(1)      , &
```

Comment:
Suggest only pass the data structure `bvarStruct%gru(iGRU)` and associate variables to this data structure in `init_summa_HDS`

Reply:
Done.